File: CaprezzoDigitale.WebApi/Models/TipoMessaggio.cs
namespace CaprezzoDigitale.WebApi.Models
{
    public class TipoMessaggio
    {
        public short Id { get; set; }
        public string DisplayName { get; set; }
        public string Descrizione { get; set; }
        public string Icona { get; set; }
        public string Colore { get; set; }
        public List<Messaggio> Messaggi { get; set; }
    }
}

File: public-client/src/app/routes/dashboard/models/tipo-messaggio.ts
export class TipoMessaggio {
  id!: number;
  displayName!: string;
  descrizione!: string;
  icona!: string;
  colore!: string;

  constructor(data?: Partial<TipoMessaggio>) {
      Object.assign(this, data);
  }
}
